lib/fetcher/fetcher_service.rb

```diff
@@ -0,0 +1,18 @@
+class FetcherService
+  def initialize(fetchers)
+    @fetchers = fetchers
+  end
+
+  def self.instance
+    FetcherService.new([V1AlphaFetcher.new])
```

Comment:
as mentioned offline, we could make this a Singleton at some point, if we start seeing this as a performance issue